resources/prometheus/unit_tests/RHACSProbeRunFailed.yaml

```diff
@@ -6,10 +6,10 @@ evaluation_interval: 1m
 tests:
   - interval: 1m
     input_series:
-      - series: acs_probe_last_failure_timestamp{namespace="rhacs-probe", pod="probe-1234"}
-        values: "0+0x10 0+0x15 7+0x60"
-      - series: acs_probe_last_success_timestamp{namespace="rhacs-probe", pod="probe-1234"}
-        values: "0+0x10 1+1x15 6+0x60"
+      - series: acs_probe_last_failure_timestamp{namespace="rhacs-probe", region="eu-west-1", rhacs_cluster_name="acs-eu-1"}
+        values: "0+0x15 17+1x60"
+      - series: acs_probe_last_success_timestamp{namespace="rhacs-probe", region="eu-west-1", rhacs_cluster_name="acs-eu-1"}
+        values: "1+1x15 17+0x60"
```

Comment:
why this has changed?

Reply:
Because the old values were actually an incorrect counter series (not monotonous). Didn't matter before, but does matter now.